services/src/main/java/org/fao/geonet/api/standards/StandardsApi.java

```diff
@@ -324,45 +327,57 @@ public String getEditorAssociatedPanelConfiguration(
         @ApiParam(value = "Schema identifier",
             required = true,
             example = "iso19139")
-        @PathVariable String schema
+        @PathVariable String schema,
+        @ApiParam(value = "Configuration identifier",
+            required = true,
+            defaultValue = "default",
+            example = "default")
+        @PathVariable String name
     ) throws Exception {
 
-        String schemaToCheck = schema;
-
-        // Store processed schemas to avoid loops
-        Set<String> schemasProcessed = new HashSet<>();
-
-        while (StringUtils.isNotEmpty(schemaToCheck) &&
```

Comment:
The idea with this loop was to find the closest dependent schema that has a configuration file,